### PR TITLE
[Feat] Updating all images to be of the same dimensions- using Cloudinary

### PR DESCRIPTION
**Pull request by twentymurial33**
Updating all images to be of the same dimensions- using Cloudinary, "Closes https://github.com/twentymurial33/murials_menu/issues/90"

- [X ] Updated all images to have w of 600 and inspected by using the elements tool

Screen Shot 2023-03-20 at 3 28 22 PM

image

### CODE REVIEW

File: client/src/pages/Details.js [hunk 1]
```diff
@@ -88,7 +94,7 @@ function Details() {
           {filteredMenuItems.map((data) => {
             return (
               <div key={data.id}>
-                <img src={data.img} alt="images" />
+                <img src={data.img} alt="images" transform={getCloudinaryURL} />
```
Comment (brentmclark) [inline client/src/pages/Details.js:97]: I don't think `transform` is a standard HTML image element property.  Did you mean to bring in a react `Img` component from a library somewhere?

Comment (brentmclark) [reply client/src/pages/Details.js:97]: If not, I think you're SUPER close to the implementation.  You'll need to replace the `src` attribute with the `getCloudinaryURL` return value.  Let's discuss it today if there's time.

File: client/src/pages/Details.js [hunk 2]
```diff
@@ -71,14 +76,15 @@ function Details() {
           {filteredMenuItems.map((data) => {
             return (
               <div key={data.id} style={{ display: "flex" }}>
+                <img src={getCloudinaryURL} alt="cloudinary" />
```
Comment (brentmclark) [inline client/src/pages/Details.js:79]: I assume this use of a functional reference does not work as expected, correct?  

Did you mean to _call_ the function here?  

```suggestion
                <img src={getCloudinaryURL(data.img)} alt="cloudinary" />
```

Comment (brentmclark) [reply client/src/pages/Details.js]: Sounds good.  Let me know when you're ready for another review.

**Comment by twentymurial33** [comment]
Question: Code resizes however when I 
 ```const getCloudinaryURL = (img) => {
    return `https://res.cloudinary.com/dac1at79b/image/upload/c_fill,w_600/${img}`;
  };
  console.log(getCloudinaryURL())```

**Pull request merged**